model-optimizer/extensions/front/onnx/loop_ext.py

```diff
@@ -105,8 +176,11 @@ def extract(cls, loop_node):
             body_results.append(Node(body_graph, add_opoutput(body_graph, node_name, output_port, False)))
 
         # add 'internal_layer_id' attribute which is a must have attribute for the loop body node
-        for idx, body_node in enumerate(body_graph.get_op_nodes()):
-            body_node['internal_layer_id'] = idx
+        cur_graph = body_graph
+        while 'parent_node' in cur_graph.graph:
+            for idx, body_node in enumerate(cur_graph.get_op_nodes()):
+                body_node['internal_layer_id'] = idx
+            cur_graph = cur_graph.graph['parent_node'].graph
```

Comment:
do we really need it since this extractor will recursively be called for every nested body graph below? I think we need to return the previous version of `for` loop